### ISSUE TITLE
[Help] How to find location of where this app is installed?

### ISSUE THREAD

**Issue by ianpan870102** [JasonStein/Notepads#626]
Normally a user can right-click on an app's icon and choose to "**Open file location**" in order to locate the .exe file of that app.

I want to find out **where Notepads is installed** (I downloaded it from Microsoft Store) so I can set it as the default app to open certain file types, since I'm supposed to **provide the path to some text editor's installation location** for it to be set as the default app.
When I right click on Notepad's icon, there's **no option** to "Open file location", so I'm having trouble locating where Notepads is exactly installed. Any help is appreaciated!

_By the way, this is one beautiful and lightweight editor. I'm enjoying it so much, and I can't believe I only started using it today!_

**Comment by MarcAnt01** [comment]
This might help you https://devblogs.microsoft.com/oldnewthing/20171120-00/?p=97425

**Comment by soumyamahunt** [comment]
> Normally a user can right-click on an app's icon and choose to "**Open file location**" in order to locate the .exe file of that app.
> When I right click on Notepad's icon, there's **no option** to "Open file location", so I'm having trouble locating where Notepads is exactly installed. Any help is appreaciated!
You can get the location of app right clicking the process in Task Manager and clicking "Open file location".

> I want to find out **where Notepads is installed** (I downloaded it from Microsoft Store) so I can set it as the default app to open certain file types, since I'm supposed to **provide the path to some text editor's installation location** for it to be set as the default app.

This is an uwp app, and you can't set uwp app as default for custom file types by that method. However, if you can I would suggest you to upgrade to Windows 10 2004, in that you will get Notepads in "Open with" menu of any type of file, and choose to set as default for custom file types.

**Comment by ianpan870102** [comment]
Thank you both for the help, I managed to get the location of the `.exe` file (though it **doesn't** carry the official Notepads icon -- but at least it **works**).

> I would suggest you to upgrade to Windows 10 2004, in that you will get Notepads in "Open with" menu of any type of file, and choose to set as default for custom file types.

I am indeed on Windows 10 2004 (latest update), however for certain file types (especially the unusual ones), Notepads does **not** show up as on of the options when I invoke "Open with..".

**Comment by soumyamahunt** [comment]
> Thank you both for the help, I managed to get the location of the `.exe` file (though it **doesn't** carry the official Notepads icon -- but at least it **works**).

It might work for some files, but for hidden files and may be some system files it won't work.

> I am indeed on **Windows 10 2004** (latest update). However, for certain file types (especially the unusual ones), Notepads does **not** show up as on of the options when I invoke "Open with..".

My bad, apparently the new changes in Notepads haven't been pushed to users. It will come in next version of Notepads.

**Comment by ianpan870102** [comment]
>  It will come in next version of Notepads.

Brilliant -- can't wait! I'll close this issue for now then.

**Comment by anyfactor** [comment]
I found mine here:

`C:\Program Files\WindowsApps\19282JackieLiu.Notepads-Beta_1.4.7.0_x64__echhpq9pdbte8`

If you go to `C:\Program Files\WindowsApps\` you will be prompted with changing security and what not. So what I did was -

1. Opened notepads
2. Opened task manager
3. Expanded the notepads section
4. Left clicked the notepads program
5. Selected properties
6. Copied the name. In my case it was - `\19282JackieLiu.Notepads-Beta_1.4.7.0_x64__echhpq9pdbte8`
7. And appended the name to  `C:\Program Files\WindowsApps\` 
8. Then in the file browser I pasted the path and you will see a `.exe` file there.
